__tests__/app.test.js

```diff
@@ -0,0 +1,42 @@
+const request = require('supertest')
+const seed = require('../db/seeds/seed')
+const app = require('../app')
+const db = require('../db/connection')
+const data = require('../db/data/test-data/index')
+
+beforeAll(() => seed(data))
+afterAll(() => db.end())
+
+describe('GET /api - controller set up correctly', () => {
```

Comment:
Unfortunately you will have to change this as the functionality of this endpoint if slightly different for this server. I do like you having considered this though